phac-nml/irida-next
# Project details UI

## What does this PR do and why?
_Describe in detail what your merge request does and why._

This PR adds in the latest activity, latest samples updated (both display last 10 items), and basic project information such as description, number of samples, number of automated workflow executions setup, number of members, and date created to the project details page. **Note: Previous change to have links take user to the project samples page by default when visiting a project have now been reverted back to the project details page**

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

![image](https://github.com/user-attachments/assets/5df3622d-d8db-4bba-9c38-037574c616c5)

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1. Create a project
2. Do a bunch of actions (Add/remove members and shared groups, add samples, update samples, attach files to samples)
3. Visit the `Details` page for the project
4. Verify that maximum last 10 activities and last 10 samples are displayed
5. Verify the information in the `Project Information` section are correct. Hovering over the icons will display what the number represents
6. Verify that all links that were pointing to the project samples page (as the default when visiting a project) now point back to the original details page

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
